lib/core/auth.js

```diff
@@ -5,6 +5,7 @@ export default class Auth {
   constructor (ctx, options) {
     this.ctx = ctx
     this.options = options
+    this._baseUrl = getProp(options, 'redirect.base') || ''
```

Comment:
We should be able to get the `router.base` option directly instead of introducting `redirect.base`